students/stachowska_agata/lesson_03_functions/collatz_sequence.py

```diff
@@ -0,0 +1,18 @@
+def collatz(x):
+    if x % 2 == 0:
+        res = x // 2
+    else:
+        res = 3 * x + 1
+    print(res)
+    return res
+
+
+while True:
+    try:
+        number = int(input("Enter a number: "))
+
```

Comment:
The exception handling should include only the part of code which might raise it

```
try:
    number = int(input("Enter a number: "))
except ValueError:
    print('You must enter an integer!')
    continue
```